lib/rbs_rails/active_record.rb

```diff
@@ -422,9 +425,12 @@ def will_save_change_to_#{col.name}?: () -> bool
             def restore_#{col.name}!: () -> void
             def clear_#{col.name}_change: () -> void
           EOS
-          sig << "attr_accessor #{col.name}? (): #{class_name}\n" if col.type == :boolean
```

Comment:
`#{col.name}?` method defined on https://github.com/rails/rails/blob/1a240cf83e03527567dea68f3c6bea5301d90113/activerecord/lib/active_record/attribute_methods/query.rb

There is no condition for this method to be defined.
Also, methods such as `model.foo? = false` are not generated. Therefore, it should not be an attr_accessor.